pkg/outputs/mysql/mysql_test.go

```diff
@@ -101,12 +102,25 @@ func TestSplitBatch(t *testing.T) {
 }
 
 func TestDDL(t *testing.T) {
-	s := `create table t(dt datetime(6) not null default current_timestamp(6) on update current_timestamp(6))`
+	s := "ALTER TABLE `mbk_redpacket`.`_mbk_redpacket_water_gho` ADD COLUMN `mt_userid` VARCHAR(25) DEFAULT NULL COMMENT '美团用户ID' AFTER `userid`, ADD INDEX `idx_mt_userid`(`mt_userid`)"
```

Comment:
Let's change the schema, table name comment to some name that looks random

Reply:
done